internal/operators/ocs/validations.go

```diff
@@ -36,6 +37,11 @@ func (o *operator) validateRequirements(cluster *models.Cluster) (api.Validation
 	hosts := cluster.Hosts
 	numAvailableHosts := int64(len(hosts))
 
+	if !validations.ValidateAdditionalNTPSource(cluster.AdditionalNtpSource) {
```

Comment:
Why do you need to validate this here? If I'm not mistaken it is already validated and populated if empty here:
https://github.com/openshift/assisted-service/blob/3f35d6cfad5156591fd76eb74dc37ffc5856a965/internal/bminventory/inventory.go#L412-L421